engine/engine/src/engine.cpp

```diff
@@ -539,7 +579,11 @@ namespace dmEngine
         char project_file[DMPATH_MAX_PATH];
         char content_root[DMPATH_MAX_PATH] = ".";
         bool loaded_ok = false;
-        if (GetProjectFile(argc, argv, project_file, sizeof(project_file)))
+
+        char res_path[DMPATH_MAX_PATH];
```

Comment:
I'm not a fan of shortening names, this is borderline hard to understand. I'd prefer "archive_path" or "resource_path".